packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -40,10 +40,20 @@ const Filter = React.forwardRef((props, ref) => {
   const applyFilterType = useListBox.types.applyFilter;
   const I18n = useI18n();
 
+  const reset = React.useCallback(() => {
+    window.requestAnimationFrame(() => {
+      applyFilter(dispatch, applyFilterType)([], false);
+      setTextSearch("");
+    });
+  }, [applyFilterType, dispatch]);
+
   React.useImperativeHandle(ref, () => ({
     clear: () => {
       setTextSearch(() => "");
     },
+    reset: () => {
+      reset();
+    },
```

Comment:
just curious can this be `reset: reset` ?

Reply:
indeed :) can.

Reply:
... or just `reset,`?

Reply:
or just  ` `! wait, that doesn't work...